homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -163,6 +164,9 @@ def _convert_forecast(self, entry):
             ATTR_FORECAST_CONDITION: self._get_condition(
                 entry.weather_code, entry.reference_time("unix")
             ),
+            ATTR_API_FORECAST_PRECIPITATION_KIND: self._calc_precipitation_kind(
```

Comment:
Only our standard forecast items are allowed in the weather forecast.

https://developers.home-assistant.io/docs/core/entity/weather#forecast